.github/workflows/daily_pull_request_automatic_update.yaml

```diff
@@ -50,3 +49,11 @@ jobs:
                     branch: ${{ matrix.actions.branch }}
                     title: '[automated] ${{ matrix.actions.name }}'
                     delete-branch: true
+
+            - name: Enable Pull Request Automerge
+                if: steps.cpr.outputs.pull-request-operation == 'created'
+                uses: peter-evans/enable-pull-request-automerge@v1
+                with:
+                token: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
it needs personal access token as documented at https://github.com/peter-evans/enable-pull-request-automerge#action-inputs